collections/_article/godot-showcase-dogwalk.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Godot Showcase - Dogwalk"
+excerpt: "Julien and Simon tell us about their experience at Blender Studio working in Dogwalk."
+categories: ["showcase"]
+author: Emi
+image: /assets/showcase/dogwalk/thumbnail.jpeg
+date: 2025-07-12 22:00:00
+---
+
+<style>
+.julien {
+	color: #aa77e2;
+}
+.simon {
+	color: #40b99f;
+}
+</style>
+
+
+Today we want to showcase a very special project. Most of you already know about it since it is really hard to go around any Godot space without finding people excited about Dogwalk. So if somehow you didn't know about the project or you still haven't been captivated by their [video blogs](https://www.youtube.com/watch?v=c4zP1sUgt6I&list=PLav47HAVZMjkxzzLDqYpTCosbsWjosUN0) over at YouTube, you are in for a treat! <strong class="julien">Julien</strong> and <strong class="simon">Simon</strong> from the [Blender Studio](https://studio.blender.org) team tell us about their experience developing their first Godot game: Dogwalk. 
+
+The game is already out on [Steam](https://store.steampowered.com/app/3775050/DOGWALK/), [Itch.io](https://blenderstudio.itch.io/dogwalk), and you can also get the project files with tons of extras from [their website](https://studio.blender.org/projects/dogwalk/). 
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/gPENs56vfYk" frameborder="0" allowfullscreen style="width: 100%; aspect-ratio: 16 / 9; height: auto;"></iframe>
+
+
+## Can you tell us a little bit about your project?
+
+<strong class="julien">Julien:</strong>
+For this year the Blender Studio, which is the in-house art department of the Blender project, decided to focus on smaller short term projects. We aimed for 4 months for each of them so I saw the unique chance to pitch a tiny game project. And the scope really had to be tiny to be able to pull off a game for the first time.
+
+Dogwalk is a short, wholesome interactive story about a dog and a kid decorating a snowman together. One big focus was on wordless storytelling through reactive gameplay. 
+It’s not strictly about fun or overcoming challenges. It’s about allowing the player to shape & experience the relationship between the two characters, simply by constraining the player to a small set of mechanics and have the game react to what they do with that.
+
+![](/assets/showcase/dogwalk/dogwalk-1.jpg)
+
+## Why did you pick Godot to develop this project?
+
+<strong class="julien">Julien:</strong>
+One of our primary missions (and constraints) is that we demonstrate how people can make entire movies by only using free open source software. We call these “Open Projects” since we also licence them as open source and make the sources available.
```

Comment:
```suggestion
One of our primary missions (and constraints) is that we demonstrate how people can make entire movies by only using free open-source software. We call these “Open Projects” since we also licence them as open source and make the sources available.
```